aikau/src/main/resources/alfresco/renderers/Thumbnail.js

```diff
@@ -465,8 +465,8 @@ define(["dojo/_base/declare",
          }
          else
          {
-            var publishGlobal = this.publishGlobal !== false;
-            var publishToParent = this.publishToParent !== false;
+            var publishGlobal = this.publishGlobal || false;
+            var publishToParent = this.publishToParent || false;
             this.alfPublish(this.publishTopic, this.publishPayload, publishGlobal, publishToParent);
```

Comment:
This boils down to personal preference... I find mentally processing multiple exclamation marks problematic as they're easily missed, what we have now is very clearly defined.